app/src/main/kotlin/com/adesso/movee/internal/injection/viewmodel/ViewModelModule.kt

```diff
@@ -14,9 +13,12 @@ import com.adesso.movee.scene.tvshow.TvShowViewModel
 import com.adesso.movee.scene.tvshowdetail.TvShowDetailViewModel
 import dagger.Binds
 import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
 import dagger.multibindings.IntoMap
 
 @Module
+@InstallIn(SingletonComponent::class)
 abstract class ViewModelModule {
```

Comment:
With hilt, we don't need to bind ViewModel classes, they are already handled. This module can be removed completely.

Reply:
Thank you for review. I missed that point. Will remove all module.

Reply:
After doing call with @muratbilir, the related code changes have been made.